src/Azure/Orleans.Streaming.EventHubs/Providers/Streams/EventHub/EventHubCheckpointerOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orleans.Configuration
+{
+    public class EventHubCheckpointerOptions
```

Comment:
These options are more specific to using azure table storage for checkpointing than they are to eventhub.  They are only related to EH because Azure table is the default checkpoint storage for the EH stream provider.  I suggests AzureTableStreamCheckpointerOptions, or something of the sort.

Reply:
sounds good